fluentd/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM fluent/fluentd:v1.6-debian-1
```

Comment:
is this container being used? if so does it need to be in the docker-compose-production file?

Reply:
I've added fluentd container to docker-compose-production.
https://github.com/antleaf/nims-hyrax/pull/300/files#diff-2013fcaf2d9b94438426326955a9d1b922017456817d18fc5888bb2b7534f43b

Reply:
The debian image is pretty hefty, (it installs LibreOffice!) but it seems to be working as intended.